yammbs/torsion/inputs.py

```diff
@@ -0,0 +1,69 @@
+from typing import Sequence
+
+import qcelemental
+from openff.qcsubmit.results import TorsionDriveResultCollection
+from pydantic import Field
+
+from yammbs._base.array import Array
+from yammbs._base.base import ImmutableModel
+
+hartree2kcalmol = qcelemental.constants.hartree2kcalmol
+bohr2angstroms = qcelemental.constants.bohr2angstroms
+
+
+class TorsionDataset(ImmutableModel):
+    tag: str
+
+
+class TorsionProfile(ImmutableModel):
+    mapped_smiles: str
+    dihedral_indices: list[int]
+
+    # TODO: Should this store more information than just the grid points and
+    #       final geometries? i.e. each point is tagged with an ID in QCArchive
+    coordinates: dict[float, Array]
```

Comment:
What units do you expect here?

Reply:
Trying to keep things internally in AKMA[^1], will document units in these models now 😅 


[^1]: though I prefer SI, I think the small molecule scientists like Angstroms)